keyboards/keebio/iris/keymaps/osiris/keymap.c

```diff
@@ -91,6 +108,18 @@ float tone_qwerty[][2]     = SONG(QWERTY_SOUND);
 
 bool process_record_user(uint16_t keycode, keyrecord_t *record) {
   switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
+        set_single_persistent_default_layer(_QWERTY);
+      }
+      return false;
```

Comment:
`break` after `return` is dead code.

If you want a keycode handled here to continue being processed by QMK (for example for One-Shot stuff), you should either `break` or `return true` in each `case`. Otherwise, just the `return false` will do.

Reply:
true. I mean false. haha..

just following the convention that's just used all across. but always a good idea to switch to the right way of doing it. let me fix it, test it and push an update. thanks

Reply:
e.g. https://github.com/qmk/qmk_firmware/blob/master/keyboards/katana60/keymaps/josefadamcik/keymap.c